pygmt/datasets/earth_age.py

```diff
@@ -67,29 +70,28 @@ def load_earth_age(
     Returns
     -------
     grid
-        The Earth seafloor crustal age grid. Coordinates are latitude and
-        longitude in degrees. Age is in millions of years (Myr).
+        The Earth seafloor crustal age grid. Coordinates are latitude and longitude in
+        degrees. Age is in millions of years (Myr).
 
     Note
     ----
     The registration and coordinate system type of the returned
-    :class:`xarray.DataArray` grid can be accessed via the GMT accessors
-    (i.e., ``grid.gmt.registration`` and ``grid.gmt.gtype`` respectively).
-    However, these properties may be lost after specific grid operations (such
-    as slicing) and will need to be manually set before passing the grid to any
-    PyGMT data processing or plotting functions. Refer to
-    :class:`pygmt.GMTDataArrayAccessor` for detailed explanations and
-    workarounds.
+    :class:`xarray.DataArray` grid can be accessed via the GMT accessors (i.e.,
+    ``grid.gmt.registration`` and ``grid.gmt.gtype`` respectively). However, these
+    properties may be lost after specific grid operations (such as slicing) and will
+    need to be manually set before passing the grid to any PyGMT data processing or
+    plotting functions. Refer to :class:`pygmt.GMTDataArrayAccessor` for detailed
+    explanations and workarounds.
 
     Examples
     --------
 
     >>> from pygmt.datasets import load_earth_age
-    >>> # load the default grid (gridline-registered 1 arc-degree grid)
+    >>> # Load the default grid (gridline-registered 1 arc-degree grid)
```

Comment:
For consitency we should also update the examples of the other datasets to start with an upper case letter for the comments.

Reply:
Done in 543a204cdbbe60e557d2daaee86bb550a8ecf60f.